src/docs/product/integrations/data-visualization/splunk/index.mdx

```diff
@@ -117,7 +117,7 @@ We’ll now begin forwarding all new events into your Splunk instance.
 
 <Note>
 
-Sentry will internally limit the maximum number of events sent to your Splunk instance to 50 per second.
+Sentry will internally limit the maximum number of events sent to your Splunk instance to 1000 per second.
```

Comment:
Is this meant to be here? Just asking since it's kind of out of context with the rest of the PR, but may as well sneak it in if it's changed.

Reply:
Yeah my bad, as part of this PR I reviewed all the docs for existing integrations to make sure I could note if they used the org slug to connect to Sentry. As part of it I noticed that this Splunk text is very out of date, and figured I'd just correct it in this same PR.